### DISCUSSION TITLE
Running Noah-MP with HYMAP2 routing

### DISCUSSION THREAD

**Discussion by cbachand** [NASA-LIS/LISF#1065] [Q&A] [answered]
Hi, I am hoping to run Noah-MP with HYMAP2 routing over a small basin in High Mountain Asia. I have a few questions about running HYMAP2. 

What does it mean to run HYMPA2 in "ensemble mode"? 

What is HYMPA2 "2-way coupling"?

I have already completed my spin-up run for Noah-MP. Is it possible to add the HYMAP2 routing to the LIS output without rerunning the LSM? Does this work for LIS "2d ensemble gridspace" output?

**Comment by agetiran** [comment, marked as answer]
Hi. 
_What does it mean to run HYMPA2 in "ensemble mode"?_
This option is commonly used for forecast runs and you have a met forcing forecast ensemble. It's useful if you are interested in streamflow simulations of each individual ensemble member.

_What is HYMPA2 "2-way coupling"?_
That option gives you the 2-way interaction between land and surface water, i.e., it represents the impacts of surface water storage (e.g., rivers, floodplains) on infiltration, soil moisture and evapotranspiration. Currently, it only works with Noah-MP 401. Please refer to  the publication below for more details.
https://agupubs.onlinelibrary.wiley.com/doi/10.1029/2021MS002463

_Is it possible to add the HYMAP2 routing to the LIS output without rerunning the LSM?_  
You don't need to rerun the whole spinup period with HyMAP. You can spinup HyMAP for only 1-2 years. We have also recently implemented options to run HyMAP using previously generated LIS-based LSM outputs that could be used as an alternative.
